BlueprintUICommonControls/Tests/Sources/UserInteractionEnabledTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UserInteractionEnabledTests.swift
+//  BlueprintUICommonControls-Unit-Tests
+//
+//  Created by Noah Blake on 3/15/20.
+//
+
+import XCTest
+import BlueprintUI
+import BlueprintUICommonControls
+
+final class UserInteractionEnabledTests: XCTestCase {
+    func test() throws {
+        let userInteractionEnabledView: (Bool) throws -> UIView = { isUserInteractionEnabled in
```

Comment:
Please let me know if there's a more concise way to get at what I'm testing here!

Reply:
This is definitely fine – another way to do this would be to wire this up to a blueprint view and inspect the created view by traversing the view hierarchy